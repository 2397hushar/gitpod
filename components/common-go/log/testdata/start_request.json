{
	"instanceId": "21473566-c15c-4ecd-9356-d1e8fdff2797",
	"level": "error",
	"message": "workspace failed",
	"serviceContext": {
		"service": "ws-manager",
		"version": "commit-13cccd83246668023ae59d02fa69911f04b17c82"
	},
	"severity": "ERROR",
	"status": {
		"id": "21473566-c15c-4ecd-9356-d1e8fdff2797",
		"status_version": 109078584033281,
		"metadata": {
			"owner": "59f72bb5-6fd4-4535-9428-9efcb84267a6",
			"meta_id": "ascend-devtools-yuzabh8j0ur",
			"started_at": {
				"seconds": 1664406817
			}
		},
		"spec": {
			"workspace_image": "eu.gcr.io/gitpod-dev/workspace-images:625c1c93fc512563a3d1592344643aed059ba3758ae77138ab8918eb8931f983",
			"deprecated_ide_image": "eu.gcr.io/gitpod-core-dev/build/ide/code:commit-97cff52e43760ad3ad96aa82d9fba934c5f53be8",
			"headless": true,
			"url": "https://ascend-devtools-yuzabh8j0ur.ws-us67.gitpod.io",
			"type": 1,
			"timeout": "30m0s",
			"ide_image": {
				"web_ref": "eu.gcr.io/gitpod-core-dev/build/ide/code:commit-97cff52e43760ad3ad96aa82d9fba934c5f53be8",
				"supervisor_ref": "eu.gcr.io/gitpod-core-dev/build/supervisor:commit-e4d1566f088bd3470ece34a281beb03bb83351b8"
			},
			"class": "default"
		},
		"phase": 6,
		"conditions": {
			"failed": "headless task failed: context canceled",
			"headless_task_failed": "headless task failed: context canceled",
			"stopped_by_request": 1,
			"volume_snapshot": {},
			"aborted": 1
		},
		"runtime": {
			"node_name": "headless-ws-us67-pool-nzrc",
			"pod_name": "prebuild-21473566-c15c-4ecd-9356-d1e8fdff2797",
			"node_ip": "10.10.0.28"
		},
		"auth": {
			"owner_token": "goXO78L43jtDkY9owZaVocHDDv08uMR8"
		}
	},
	"time": "2022-09-28T23:18:25Z",
	"userId": "59f72bb5-6fd4-4535-9428-9efcb84267a6",
	"workspaceId": "ascend-devtools-yuzabh8j0ur"
}