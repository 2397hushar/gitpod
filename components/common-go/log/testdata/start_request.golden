{
    "level": "panic",
    "msg": "",
    "req": {
        "instanceId": "21473566-c15c-4ecd-9356-d1e8fdff2797",
        "level": "error",
        "message": "workspace failed",
        "serviceContext": {
            "service": "ws-manager",
            "version": "commit-13cccd83246668023ae59d02fa69911f04b17c82"
        },
        "severity": "ERROR",
        "status": {
            "auth": {
                "owner_token": "[REDACTED]"
            },
            "conditions": {
                "aborted": 1,
                "failed": "headless task failed: context canceled",
                "headless_task_failed": "headless task failed: context canceled",
                "stopped_by_request": 1,
                "volume_snapshot": {}
            },
            "id": "21473566-c15c-4ecd-9356-d1e8fdff2797",
            "metadata": {
                "meta_id": "ascend-devtools-yuzabh8j0ur",
                "owner": "59f72bb5-6fd4-4535-9428-9efcb84267a6",
                "started_at": {
                    "seconds": 1664406817
                }
            },
            "phase": 6,
            "runtime": {
                "node_ip": "10.10.0.28",
                "node_name": "headless-ws-us67-pool-nzrc",
                "pod_name": "prebuild-21473566-c15c-4ecd-9356-d1e8fdff2797"
            },
            "spec": {
                "class": "default",
                "deprecated_ide_image": "eu.gcr.io/gitpod-core-dev/build/ide/code:commit-97cff52e43760ad3ad96aa82d9fba934c5f53be8",
                "headless": true,
                "ide_image": {
                    "supervisor_ref": "eu.gcr.io/gitpod-core-dev/build/supervisor:commit-e4d1566f088bd3470ece34a281beb03bb83351b8",
                    "web_ref": "eu.gcr.io/gitpod-core-dev/build/ide/code:commit-97cff52e43760ad3ad96aa82d9fba934c5f53be8"
                },
                "timeout": "30m0s",
                "type": 1,
                "url": "https://ascend-devtools-yuzabh8j0ur.ws-us67.gitpod.io",
                "workspace_image": "eu.gcr.io/gitpod-dev/workspace-images:625c1c93fc512563a3d1592344643aed059ba3758ae77138ab8918eb8931f983"
            },
            "status_version": 109078584033281
        },
        "time": "2022-09-28T23:18:25Z",
        "userId": "59f72bb5-6fd4-4535-9428-9efcb84267a6",
        "workspaceId": "ascend-devtools-yuzabh8j0ur"
    },
    "time": "0001-01-01T00:00:00Z"
}